map.js

```diff
@@ -11,70 +17,86 @@ let mapY = 0;
 
 let mousedown = false;
 
-let debug = true; //makes iframe and guide small by default for map function testing.
+let img_Map;
+
+let icon_Ayleid;
+let icon_Camp;
+let icon_Cave;
+let icon_Fort;
+let icon_Gate;
+let icon_Inn;
+let icon_Settlement;
+let icon_Mine; //still needs image
+let icon_Landmark; //still needs image
+let icon_Shrine; //still needs image
+
+let locArr;
+
+let debug = false; //makes iframe and guide small by default for map function testing.
 
 function initMap(){
+    //load map cord data
+    fetch("data/locations.json").then(response => response.json()).then(response => locArr = response.elements);
+
     canvas = document.getElementById("canvas_Map");
     mapCanvasContext = canvas.getContext("2d");
     
-    img = document.createElement("img");
-    img.width = 3544;
-    img.height = 2895;
-    img.src = "images/Cyrodil_Upscaled.png";
-    img.onload = drawMap();
-
+    initImgs();
+        
     //center map on imp city
-    mapX = img.width/2.1;
-    mapY = img.height/3.2;
+    mapX = img_Map.width/2.1;
+    mapY = img_Map.height/3.2;
 
+    //Input listeners
     wrapper = document.getElementById("wrapper_Map");
     wrapper.onmousedown = function(){mousedown = true;};
     wrapper.onmouseup = function(){mousedown = false;};
     wrapper.onmouseout = function(){mousedown = false;};
-    wrapper.onmousemove = function(e){
-        if(mousedown){
-            moveMap(e);
-        }
-    };
+    wrapper.onmousemove = function(e){if(mousedown){moveMap(e);}};
     wrapper.onwheel = function(e){zoomMap(e)};
 
     //attaches width to width of iframe
     window.addEventListener("mousemove", iFrameCheck);
```

Comment:
does this need to be on window? or do elements get mousemove events sent to them if mouse is over them?

Reply:
the wrapper is collecting inputs fine so far. If it's placed on window, it will pick up from the regular guide area.

Reply:
we don't want it to pick up from the regular guide area, right? so why is this attached to window?
eh whatever it'll be refactored later when we make map async